packages/mdc-menu/foundation.ts

```diff
@@ -134,12 +133,15 @@ export class MDCMenuFoundation extends MDCFoundation<MDCMenuAdapter> {
   /**
    * Selects the list item at `index` witnin the specified selection group.
    * @param index Index of list item within the selection group.
-   * @param selectionGroupIndex Index of selection group within the menu.
    */
-  setSelectedIndex(index: number, selectionGroupIndex = 0) {
-    const selectionGroup = this.adapter_.getSelectionGroupAtIndex(selectionGroupIndex);
+  setSelectedIndex(index: number) {
+    const listItem = this.adapter_.getListItemByIndex(index);
+    if (!listItem) {
+      throw new Error('No list item at specified index.');
```

Comment:
Add `MDCMenuFoundation: ` at the beginning of the error message. This'll help easily debug the error. Same below.